### ISSUE TITLE
Different casing for maps

### ISSUE THREAD

**Issue by leoshusar** [Tienisto/flutter-fast-i18n#38]
I think it would be good feature to have the possibility to not apply key_case to maps. Or select different casing for maps. Or select different casing for each map (but I would be happy with global maps setting).
I am using `key_case: camel` for all of my translations, because it's the Dart's language style.
My API returns JSON error response in format
```json
{
    "code": "user_not_found",
    "reason": "User was not found."
}
```
and now I would like to use that code to lookup the translated reason message.
Something like
```dart
try {
  something();
} on ApiException catch (ex) {
  _notificationService.showError(t.apiCodes[ex.code]);
}
```
but since all keys are now `camelCase`, I would need to do my own recasing of API response codes to `camelCase`. Which is not a big deal but this feature would still be great :)

**Comment by Tienisto** [comment]
Thanks for this issue. I will look for a global setting :)

**Comment by Tienisto** [comment]
Released in `5.2.0`
